lib/components/serverside-data/ServersideIterator.vue

```diff
@@ -72,9 +72,6 @@ export default {
     itemsPerPageDefault: {
       type: Number,
       default: () => 50,
-      validator: (v) => {
-        return defaultItemsPerPageOptions.includes(v)
-      }
     },
```

Comment:
Could you explain that a little bit more? I don't see the point.

Reply:
As discussed, this validator would fail if the user of the component would pass in an `itemsPerPageOptions` array that does not contain any of the numbers contained in `defaultItemsPerPageOptions`.